docs/src/main/asciidoc/security.adoc

```diff
@@ -39,8 +39,7 @@ Please see xref:security-built-in-authentication.adoc#basic-auth[Basic HTTP Auth
 === WebAuthn Authentication Mechanism
 
 https://webauthn.guide/[WebAuthn] is an authentication mechanism designed to replace passwords. In short, every
-time you write a service for registering new users, or logging them in, instead of asking for a password, you
-use WebAuthn, which will replace the password.
+time you write a service for registering new users, or logging them in, instead of asking for a password, you use WebAuthn, which will replace the password.
```

Comment:
Two choices:
One sentence per line, 
or a semantic chunk per line 
(which, in this case, would mean breaking at a comma instead of in the middle of the sentence).